modules/connectivity/ConnectionQuality.js

```diff
@@ -385,42 +332,22 @@ export default class ConnectionQuality {
         } else {
             // Calculate a value based on the sending bitrate.
 
-            // Figure out if simulcast is in use
+            // Figure out if simulcast is in use.
             const activeTPC = this._conference.getActivePeerConnection();
-            const isSimulcastOn
-                = Boolean(activeTPC && activeTPC.isSimulcastOn());
-
-            const newVideoBitrateCap
-                = activeTPC && activeTPC.bandwidthLimiter
-                && activeTPC.bandwidthLimiter.getBandwidthLimit('video');
+            const isSimulcastOn = Boolean(activeTPC && activeTPC.isSimulcastOn());
+            const videoQualitySettings = activeTPC.getTargetBitrates();
```

Comment:
What if activeTPC is undefined? You protected yourself from it in the previous line but not here.

Reply:
Good point, will fix it.